tests/k8s/curl.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: curl
+  labels:
+    name: curl
+spec:
+  containers:
+  - name: curl
+    image: byrnedo/alpine-curl
+    command:
+      - "sh"
+      - "-c"
+      - >
+        while true; do
+          sleep 3600;
```

Comment:
why is this needed?

Reply:
To keep the container running by docker engine